graphql2/graphqlapp/calendarsubscription.go

```diff
@@ -0,0 +1,98 @@
+package graphqlapp
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"github.com/target/goalert/calendarsubscription"
+	"github.com/target/goalert/graphql2"
+	"github.com/target/goalert/schedule"
+)
+
+type CalendarSubscription App
+
+func (a *App) CalendarSubscription() graphql2.CalendarSubscriptionResolver {
+	return (*CalendarSubscription)(a)
+}
+
+func (a *CalendarSubscription) ReminderMinutes(ctx context.Context, obj *calendarsubscription.CalendarSubscription) ([]int, error) {
+	var config calendarsubscription.Config
+	err := json.Unmarshal(obj.Config, &config)
+	obj.ReminderMinutes = config.ReminderMinutes
+	return obj.ReminderMinutes, err
+}
+func (a *CalendarSubscription) ScheduleID(ctx context.Context, obj *calendarsubscription.CalendarSubscription) (string, error) {
+	e := *obj
+	return e.ScheduleID, nil
+}
+func (a *CalendarSubscription) Schedule(ctx context.Context, obj *calendarsubscription.CalendarSubscription) (*schedule.Schedule, error) {
+	return a.ScheduleStore.FindOne(ctx, obj.ScheduleID)
+}
+func (a *CalendarSubscription) URL(ctx context.Context, obj *calendarsubscription.CalendarSubscription) (*string, error) {
+	var err error
+	return nil, err
```

Comment:
We should return an error, stub URL, or omit this field entirely from this PR. Probably the latter.